### PR TITLE
Update DevConf.US FAQs

### PR DESCRIPTION
**Pull request by umohnani8**
Update the FAQs with more information on videos.

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

File: build_production/us/faq/index.html [hunk 1]
```diff
@@ -240,11 +252,59 @@
...
 				<div class="card-header collapsed mont-600" id="heading-speaker1" data-toggle="collapse" 
 				     data-target="#collapse-speaker1" aria-expanded="true" aria-controls="collapse-speaker1">
-						What else should I know?						<i class="fa"></i>
+						Are there any video recording guidelines?						<i class="fa"></i>
 					
 				</div>
 
 				<div id="collapse-speaker1" class="collapse" aria-labelledby="heading-speaker1" data-parent="#sub-speaker">
+					<div class="card-body os-400">
+						Yes! We have put together steps on how to record an awesome video with the opens source software OBS. The steps can be found <a href='https://github.com/devconfcz/virtual-presentations/blob/master/Recording.md'>here</a>.					</div>
```
Comment (whitel) [inline build_production/us/faq/index.html:261]: typo: "open source"

File: build_production/us/faq/index.html [hunk 2]
```diff
@@ -240,11 +252,59 @@
...
+					<div class="card">
+				<div class="card-header collapsed mont-600" id="heading-speaker3" data-toggle="collapse" 
+				     data-target="#collapse-speaker3" aria-expanded="true" aria-controls="collapse-speaker3">
+						How should I submit the pre-recorded video?						<i class="fa"></i>
+					
+				</div>
+
+				<div id="collapse-speaker3" class="collapse" aria-labelledby="heading-speaker3" data-parent="#sub-speaker">
+					<div class="card-body os-400">
+						We recommend uploading the video to youtube as unlisted. We will send out an email with instructions on how to submit the youtube link of your video. <br><br> If you would prefer not to use youtube, send the organizers an email at <a href='mailto:info@devconf.us'>info@devconf.us</a> and we will work with you to figure out alternate options.					</div>
```
Comment (whitel) [inline build_production/us/faq/index.html:285]: YouTube

File: build_production/us/faq/index.html [hunk 3]
```diff
@@ -240,11 +252,59 @@
...
+					<div class="card">
+				<div class="card-header collapsed mont-600" id="heading-speaker4" data-toggle="collapse" 
+				     data-target="#collapse-speaker4" aria-expanded="true" aria-controls="collapse-speaker4">
+						Will there be speaker coaching?						<i class="fa"></i>
+					
+				</div>
+
+				<div id="collapse-speaker4" class="collapse" aria-labelledby="heading-speaker4" data-parent="#sub-speaker">
+					<div class="card-body os-400">
+						Yes! When you confirm your session acceptance, you will be asked whether you would like to have a speaker coach and will be matched with one.					</div>
```
Comment (whitel) [inline build_production/us/faq/index.html:297]: are we actually asking?

File: source/us/faq.blade.php [hunk 4]
```diff
@@ -77,8 +79,13 @@
 		array("group" => "Speaker Questions", "id" => "speaker", 
 			"questions" => array( 
 				array("q" => "Why does the speaker confirmation form ask you for a sched.org email address?", "a" => "We will be publishing our schedule there and having your preferred email address for that service lets us give you some editing rights for your session. Some speakers may use a different address for sched.org so we want to collect that from them. If you don’t have a preference or don’t have one already, just provide any email address you read and we will do the rest."),
-				array("q" => "What else should I know?", "a" => "We're excited for our first Virtual DevConf.US.  We hope you'll be patient with us as we work out the details, together. Please email us at <a href='mailto:info@devconf.us'>info@devconf.us</a> with any other questions and especially any ideas you have!")
-		)),
+				array("q" => "Are there any video recording guidelines?", "a" => "Yes! We have put together steps on how to record an awesome video with the opens source software OBS. The steps can be found <a href='https://github.com/devconfcz/virtual-presentations/blob/master/Recording.md'>here</a>."),
```
Comment (whitel) [inline source/us/faq.blade.php:82]: same typo

File: build_production/us/faq/index.html (hunk 3 above)
Comment (umohnani8) [reply build_production/us/faq/index.html:297]: Yes we are, it is in the confirmation form.

File: source/us/faq.blade.php (hunk 4 above)
Comment (umohnani8) [reply source/us/faq.blade.php]: Fixed

File: build_production/us/faq/index.html (hunk 2 above)
Comment (umohnani8) [reply build_production/us/faq/index.html]: Fixed

File: build_production/us/faq/index.html (hunk 1 above)
Comment (umohnani8) [reply build_production/us/faq/index.html]: Fixed

**Comment by sallyom** [comment]
/lgtm 

Thanks @umohnani8!

**Comment by umohnani8** [comment]
@jridky PTAL :)

**Pull request merged**